common/prometheus-server/templates/thanos/prometheus-thanosQueryDeployment.yaml

```diff
@@ -33,6 +33,7 @@ spec:
         - --log.level={{ .Values.logLevel }}
         - --query.replica-label=prometheus_replica
         - --query.auto-downsampling
+        - --query.timeout={{ include ".Values.thanos.querier.timeout" }}:2m
```

Comment:
Is `:2m` is intended? Why `include`?

Reply:
The intention was if a value was not provided, to default to 2m, which is the regular default. The point was to allow it to be configurable, but not change the default. I thought include was the proper way to do that in this syntax.

Reply:
I get that. Sorry if I wasn't clear: This is technically not working as you intended. Values are not `included` like that and defaults, if that's what the `:2m` should be for, are not provided like that.